File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Ocsa/OCS0103U00LayCommonInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Ocsa
{
    [Serializable]
    public class OCS0103U00LayCommonInfo
    {
        private String _name;
        private String _bulyong;
        private String _drgYn;

        public String Name
        {
            get { return this._name; }
            set { this._name = value; }
        }

        public String Bulyong
        {
            get { return this._bulyong; }
            set { this._bulyong = value; }
        }

        public String DrgYn
        {
            get { return this._drgYn; }
            set { this._drgYn = value; }
        }

        public OCS0103U00LayCommonInfo() { }

        public OCS0103U00LayCommonInfo(String name, String bulyong, String drgYn)
        {
            this._name = name;
            this._bulyong = bulyong;
            this._drgYn = drgYn;
        }

    }
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/ocsa/OCS0103U00LayCommonInfo.java
package nta.med.data.model.ihis.ocsa;

import java.io.Serializable;
import java.util.Date;

public class OCS0103U00LayCommonInfo implements Serializable {
	private String name          ; 
	private Date bulyong       ; 
	private String drgYn         ;
	public OCS0103U00LayCommonInfo(String name, Date bulyong, String drgYn) {
		super();
		this.name = name;
		this.bulyong = bulyong;
		this.drgYn = drgYn;
	}
	public String getName() {
		return name;
	}
	public void setName(String name) {
		this.name = name;
	}
	public Date getBulyong() {
		return bulyong;
	}
	public void setBulyong(Date bulyong) {
		this.bulyong = bulyong;
	}
	public String getDrgYn() {
		return drgYn;
	}
	public void setDrgYn(String drgYn) {
		this.drgYn = drgYn;
	}
}
